test/e2e/storage/pv_protection.go

```diff
@@ -83,9 +84,7 @@ var _ = utils.SIGDescribe("PV Protection", func() {
 		framework.ExpectNoError(framework.WaitForPersistentVolumePhase(v1.VolumeAvailable, client, pv.Name, 1*time.Second, 30*time.Second))
 
 		By("Checking that PV Protection finalizer is set")
-		pv, err = client.CoreV1().PersistentVolumes().Get(pv.Name, metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred(), "While getting PV status")
-		Expect(slice.ContainsString(pv.ObjectMeta.Finalizers, volumeutil.PVProtectionFinalizer, nil)).To(BeTrue(), "PV Protection finalizer(%v) is not set in %v", volumeutil.PVProtectionFinalizer, pv.ObjectMeta.Finalizers)
+		check_pv_protection_finalizer(client, pv)
```

Comment:
typical pattern we use in these tests is:

```
err = checkForPVProtectionFinalizer(...)
Expect(err).NotTo(HaveOccured(), "while waiting for finalizer to be added")
```

Reply:
Thanks again, will apply that.